app/controllers/issues_controller.rb

```diff
@@ -23,8 +23,10 @@ def followees
   def create
     @issue = Issue.new(issues_params)
     @issue.user_id = current_user.id
+    @user = current_user
     if @issue.save
-      redirect_to root_path, notice: 'Звернення створене успішно!'
+      IssueMailer.issue_created(@issue, @user).deliver
```

Comment:
create method notify support in issue.rb and send email from it after issue created - now from controller